website/pages/docs/agent/options.mdx

```diff
@@ -2124,6 +2133,17 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
       the proxy will prevent any access to paths without that prefix on the
       backend.
 
+    - `path_allowlist` ((#ui_config_metrics_provider_path_allowlist)) - This
```

Comment:
So over in the TODO PR you indicated this should be `allowed_paths` instead. Which one should it be?

Reply:
I used this in a bunch of places - just that one bit inline in the ui viz page is wrong so I'll fix that for now. I think this is more consistent with out other `*_allowlist` fields.